R/install-remote.R

```diff
@@ -166,6 +166,11 @@ package2remote <- function(name, lib = .libPaths(), repos = getOption("repos"),
   }
 
   switch(x$RemoteType,
+    standard = remote("cran",
```

Comment:
Well, I don't know if this would cause problems here, but "standard" just means that it was coming from a CRAN-like repo.

I think remotes will be fine with `"cran"`, though....

Reply:
I figured this was the easiest way (at least for now) to have `remotes` just treat these standard remote types as CRAN-like, since `cran` remotes are already well-understood and handled elsewhere in the system. Do you think this should be handled differently?

At least, I believe this works as a stop-gap solution.

Reply:
I _think_ it will be fine. I think for remotes "cran" means "cran-like", mostly.

Reply:
@jimhester do you think this would fail in any way?

Reply:
Yeah I think this would be fine.